src/front/spv/mod.rs

```diff
@@ -1754,6 +1758,16 @@ impl<I: Iterator<Item = u32>> Parser<I> {
             }?;
         }
 
+        log::info!("Patching...");
+        // patch all the function calls
+        for (_, fun) in module.functions.iter_mut() {
+            self.patch_function_calls(fun)?;
+        }
+        for ep in module.entry_points.iter_mut() {
+            self.patch_function_calls(&mut ep.function)?;
+        }
+        self.lookup_function.clear();
```

Comment:
I always wondered why there are `clear` function calls used. Is this better that reinitializing the collection?